libs/saveFiles.js

```diff
@@ -124,7 +131,7 @@ const saveEntry = function (entry, options) {
     })
     .catch(err => {
       log('warn', errors.FILE_DOWNLOAD_FAILED)
-      log('warn', err.message, `Error caught while trying to save the file ${entry.fileurl}`)
+      log('warn', err.message, `Error caught while trying to save the file ${entry.fileurl ? entry.fileurl : entry.filename}`)
```

Comment:
SAVE_FILE_FAILED ?

Reply:
Instead of the `FILE_DOWNLOAD_FAILED`?

Reply:
Let's go for SAVE_FILE_FAILED. This error code is not used anywhere else at the moment. Then we can change it freely.

Reply:
Done